lessons/module-2/objects-and-prototype-chain.md

```diff
@@ -43,37 +43,37 @@ In JavaScript, a prototype is a special object that is assigned to all functions
 Take a look at the example below:
 
 ```js
-class Instructor {
-  constructor(name, primaryLesson) {
-    this.name = name;
-    this.primaryLesson = primaryLesson;
-  }
-
-  teachLesson(duration) {
-    if (duration > 3) {
-      return `${this.name} can\'t teach a lesson that long!`;
-    } else {
-      return `Gunna teach you all real good about ${this.primaryLesson}`;
-    }
-  }
-
-  gradeProjects(project) {
-    return `${this.name} is grading ${project}.`
+const createInstructor = (name, primaryLesson) => {
+  return {
+    name: name,
+    primaryLesson: primaryLesson,
+    teachLesson: (duration) => {
+      if (duration > 3) {
+        return `${this.name} can\'t teach a lesson that long!`
+      } else {
+        return `Gunna teach you all real good about ${this.primaryLesson}`
+      }
+    },
+    gradeProjects: (project) => {
+      return `${this.name} is grading ${project}.`
+    } 
   }
 }
 
-let instructor = new Instructor('Hannah', 'OOP 2');
+const nick = createInstructor('Nick', 'Intro to Accessibility')
 ```
 
-* After declaring this class, type `Instructor.` into your console.  (*Note the uppercase `I`*) You should see the browser attempting to autocomplete this with properties that are available on this function.  Look at the `prototype` property and take note of its value.
+* After declaring this function and initializing the variable `nick`, type `nick.` into your console. You should see the browser attempting to autocomplete this with properties that are available on this function.  Look at the `nick.constructor.prototype` property and take note of its value.
 </section>
 
 <section class="answer">
 ### Diving Deeper  
 
-By accessing the `prototype` of our `Instructor` class - we can see that it points to an object that has three methods: `constructor`, `teachLesson`, and `gradeProject`.  Our `prototype` is a property on the function that points back to our `Instructor` class. 
+Let's consider the `Array` data type in JavaScript. In your console, access the `prototype` of `Array` - what do you notice? The `constructor` of all arrays defines the methods on our `Array.prototype` - you might be familiar with some of these array prototype methods already!
```

Comment:
This is a great example to help them make connections to things they are already using!